src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -16,6 +16,7 @@
  */
 
 #include "ObjectMgr.h"
+#include "AchievementMgr.h"
```

Comment:
shouldnt this be `CriteriaHandler.h` (thats where `sCriteriaMgr` lives)

Reply:
Yep, you're right. Fixed!